app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt

```diff
@@ -1,34 +1,116 @@
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
 import no.usn.mob3000.R
 import no.usn.mob3000.Viewport
 
+
 /**
- * This is the chessboard page, where users can free-play against an AI or
- * fellow physically near player. (Local multiplayer, in other words).
+ * This is the login screen for authorization to an account
+ * as well as an option for logging in with a guest account.
  *
  * @see ForgotPasswordScreen
  * @see ResetPasswordScreen
- * @author frigvid
- * @created 2024-09-24
+ * @author Anarox
+ * @created 2024-09-30
  */
 @Composable
-fun LoginScreen() {
+fun LoginScreen(
+    onLogin: () -> Unit,
+    onForgot: () -> Unit
+) {
+    var username by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
     Viewport { innerPadding ->
         Box(
-            modifier = Modifier.fillMaxSize()
-                               .padding(innerPadding),
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(innerPadding),
             contentAlignment = Alignment.Center
         ) {
-            Text(stringResource(R.string.auth_login_title))
+            Column(
+                modifier = Modifier.padding(16.dp),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                // Title
+                Text(text = stringResource(R.string.auth_login_title))
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                // Username input
+                TextField(
+                    value = username,
+                    onValueChange = { username = it },
+                    label = { Text("Brukernavn") },
+                    modifier = Modifier.fillMaxWidth()
+                )
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                // Password input
+                TextField(
+                    value = password,
+                    onValueChange = { password = it },
+                    label = { Text("Passord") },
```

Comment:
String resources should be extracted. And localized. Do not mix-and-match, be consistent. There's both English and Norwegian here, when there should only be one, or be using localized string resources.